create_jenkins_job.py

```diff
@@ -163,22 +177,26 @@ def main(argv=None):
             job_data['enable_c_coverage_default'] = 'false'
 
         # configure nightly triggered job
-        job_name = 'nightly_' + os_name + '_release'
-        if os_name == 'win':
-            job_name = job_name[0:-4]
-        job_data['cmake_build_type'] = 'Release'
-        job_config = expand_template('ci_job.xml.em', job_data)
-        configure_job(jenkins, job_name, job_config, **jenkins_kwargs)
+        # no aarch64 Release job yet
+        if os_name != 'linux-aarch64':
+            job_name = 'nightly_' + os_name + '_release'
+            if os_name == 'win':
+                job_name = job_name[0:-4]
+            job_data['cmake_build_type'] = 'Release'
+            job_config = expand_template('ci_job.xml.em', job_data)
+            configure_job(jenkins, job_name, job_config, **jenkins_kwargs)
 
         # configure nightly triggered job with repeated testing
-        job_name = 'nightly_' + os_name + '_repeated'
-        if os_name == 'win':
-            job_name = job_name[0:-5]
-        job_data['time_trigger_spec'] = '0 12 * * *'
-        job_data['cmake_build_type'] = 'None'
-        job_data['ament_test_args_default'] = '--retest-until-fail 20 --ctest-args -LE linter --'
-        job_config = expand_template('ci_job.xml.em', job_data)
-        configure_job(jenkins, job_name, job_config, **jenkins_kwargs)
+        # no aarch64 repeated job yet
+        if os_name != 'linux-aarch64':
+            job_name = 'nightly_' + os_name + '_repeated'
+            if os_name == 'win':
+                job_name = job_name[0:-5]
+            job_data['time_trigger_spec'] = '0 12 * * *'
```

Comment:
This time is likely going to collide with the periodic reboot of the router. So it might be better to wait with this until the conflict is resolved.